tools/serverpod_cli/lib/src/migrations/generator.dart

```diff
@@ -66,6 +69,7 @@ class MigrationGenerator {
     );
     try {
       return await MigrationVersion.load(
+        moduleName: module,
         versionName: versionName,
         migrationsDirectory: migrationsDirectory,
```

Comment:
Suggestion(Non-blocking): Move the migrationsDirectory construction into the load method.

Since we now pass in the module name as well we might as well do the construction for the migrations directory inside of `MigrationVersion.load()`.

Reply:
Not sure I follow here.

Reply:
On the lines above we create the migrationsDirectory. But we have all the information to do this internally now inside of the load method. We could just pass in the migrationBaseDirectory and then MigrationVersion could construct the path to the designated directory.

(The first thing we do inside of `load(...)` is actually to build another directory path using the version name. So the building on the lines above here is almost completetly redundant.)